src/app/components/cron-gen/cron-gen.jsx

```diff
@@ -109,59 +128,66 @@ function ChplCronGen(props) {
   useEffect(() => updateCron(), [days, formik.values.runTime]);
 
   return (
-    <div className={classes.content}>
+    <div>
```

Comment:
Is there a reason for this `<div>` any more? If it doesn't apply a class, and the only thing inside it is the `<Card>` element, then I don't think it provides any value, and instead just adds a level of nesting that's not necessary

Reply:
Need the extra div so the layout get applied correctly